README.md

```diff
@@ -13,6 +13,6 @@ This is a template repo used for creating new packages for `stactools`.
 3. Update `setup.cfg` with your package description and such.
 4. Update the LICENSE with your company's information (or whomever holds the copyright).
 5. Run `sphinx-quickstart` in the `docs` directory to create the documentation template.
-6. Update `docs/installation_and_basic_usage.ipynb` to provide an interactive notebook to help users get started. Include the following badge at the top of the README to launch the notebook: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/stactools-packages/template/main?filepath=docs/installation_and_basic_usage.ipynb). Be sure to modify the badge href to match your package repo.
+6. Update `docs/installation_and_basic_usage.ipynb` to provide an interactive notebook to help users get started.
```

Comment:
What's the reason we're removing this?

Reply:
My intention with these changes was to have the notebook run in the Docker container by default, so it wouldn't require any packages to be installed. I realised that meant the notebook wouldn't work on binder.org. Hindsight, leaving a few package install commands in the notebook to allow for greater portability (while still working in the container) was the better option.